concept/answer/BUILD

```diff
@@ -25,12 +25,14 @@ java_library(
     name = "answer",
     srcs = glob(["*.java"]),
     deps = [
+        "//concept/util:util",
         "//kb/concept/api:api",
 
         # External dependenicies from @graknlabs
         "@graknlabs_graql//java:graql",
 
         # External dependencies from Maven
+        "//dependencies/maven/artifacts/com/google/guava",
```

Comment:
This is minor, but I think the idea was to start collecting `Collection`s and utils in our `common` package so that we don't have to depend on Guava for trivial things, like `Sets.newHashSet()` instead of the native `new HashSet()`, also functions like `intersection` I am sure can be copied over in common.
Again, this is just a comment for the future, but ideally if we are not using fancy Guava functionalities, at some point  we could stop depending on it. But ofc it's a long term thing, because some of our deps still depend on Guava 😄